portfolio/src/test/java/com/google/sps/LinkedListTimeRangeGroupTest.java

```diff
@@ -0,0 +1,557 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class LinkedListTimeRangeGroupTest {
```

Comment:
Consider using a parameterized test, which runs the same test suite on LinkedListTimeRangeGroup as well as ArrayListTimeRangeGroup.

Reference:
https://junit.org/junit4/javadoc/4.12/org/junit/runners/Parameterized.html

Maybe we can name it as TimeRangeGroupTest

Reply:
Done.